rocks/RocksStorage.java

```diff
@@ -166,11 +165,13 @@ public Cache(OptimisticTransactionDB rocksDB) {
         }
 
         @Override
-        public byte[] get(byte[] key) {
+        public ByteArray get(ByteArray key) {
             try {
                 deleteCloseSchemaWriteLock.readLock().lock();
                 if (!isOpen()) throw TypeDBException.of(RESOURCE_CLOSED);
-                return storageTransaction.get(readOptions, key);
+                byte[] value = storageTransaction.get(readOptions, key.getBytes());
+                if (value == null) return null;
+                else return ByteArray.of(value);
```

Comment:
we now have this pattern when we do `get` to return a null, otherwise the `byte[]` inside the `ByteArray()` would be null, which doesn't look like a nonexistent key or value, when it really is